src/App.vue

```diff
@@ -17,20 +17,21 @@
             </div>
             <div id="navbarMenuHeroA" class="navbar-menu">
               <div class="navbar-end">
-                <a class="navbar-item is-active">
+                <a class="navbar-item is-active" href="https://pangeo-forge.org">
                   Home
                 </a>
-                <a class="navbar-item">
+                <a class="navbar-item" href="https://pangeo-forge.readthedocs.io/en/latest/recipe_box.html">
                   Recipes
                 </a>
                 <a class="navbar-item">
                   Catalog
                 </a>
-                <a class="navbar-item">
+                <a class="navbar-item" href="https://pangeo-forge.readthedocs.io/en/latest/">
```

Comment:
It would be great to eventually have docs for the project as a whole. These docs are just for the `pangeo_forge_recipes` python package. Maybe that's the same thing? Still unsure how to sort this all out.

Reply:
Yes, 💯 % agree greater clarity here can be a point of improvement for us. I suppose the current docs *do* touch on the notion of Bakeries and other non-recipe-specific topics, which makes me think they are more general than `pangeo_forge_recipes` alone.